corehq/form_processor/backends/couch/dbaccessors.py

```diff
@@ -126,6 +127,14 @@ def _form_undelete(doc):
             doc['server_modified_on'] = json_format_datetime(datetime.utcnow())
         return _soft_undelete(XFormInstance.get_db(), form_ids, _form_undelete)
 
+    @staticmethod
+    def modify_attachment_xml_and_metadata(form_data, form_attachment_new_xml):
```

Comment:
this still needs to modify the form metadata for couch forms

Reply:
Fixed across these 2 commits:

https://github.com/dimagi/commcare-hq/pull/19979/commits/39575949e103611d51e4b0a962ffb7528dda8467

https://github.com/dimagi/commcare-hq/pull/19979/commits/856c22edd5a2e6a8def96fa90f4510f04da6754d